src/types.ts

```diff
@@ -245,3 +277,13 @@ export interface IEventListener {
   internal: boolean;
   callback(evt: TAllEvents): any;
 }
+
+export type TOnRequestCryptoAccountCallback = (
+  type: string,
+  assetSymbol: string
+) => Promise<{
```

Comment:
Wouldn't it make sense to extract the return type of this promise to a separate type? You're using it in 3 places anyway 🤔